packages/website/src/components/SplitPaneResize.css

```diff
@@ -0,0 +1,49 @@
+.Resizer {
+  background: #000;
```

Comment:
This is going to need to use CSS variables for color values the way other CSS files in the website do. We synchronize the color theme using them, and also they make dark mode support a lot easier. Which is a bug right now 😄. You can't see the resizers in dark mode:

![Screenshot of dark mode grid resizing in the playground with not-visible black-on-black resizers](https://user-images.githubusercontent.com/3335181/191346514-bb80f24a-c7f9-4e14-aecd-f72aad7f0cde.png)

Reply:
Updated to use CSS variables. 
![image](https://user-images.githubusercontent.com/46636043/191380577-2a9a6730-ce82-4294-bfb5-d6c71a69e13d.png)